docs/module_docs/GNPS_export/gnps_export.md

```diff
@@ -0,0 +1,74 @@
+# GNPS export

```

Comment:
maybe as a title and in the navigarion 
GNPS - Feature-based Molecular Networking (FBMN)